CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing Overview
```

Comment:
Can we specify that this contributing guide works for WasmEdge/docs repo?

Reply:
Done 💯